core/services/synchronization/explorer_client_test.go

```diff
@@ -18,25 +18,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestWebSocketClient_StartCloseStart(t *testing.T) {
```

Comment:
From what I could tell, the `explorerClient` is only started once inside `chainlink.Application` and never restarted.